### PR TITLE
octopus: mgr/dashboard: Update translations on octopus

### PR DESCRIPTION
**Pull request by Exotelis**
This commit is going to update the translations for octopus.

This manual backport is not a cherry-pick because we don't want that the
changes of the translations as well as the changes in the
i18n.config.json take affect on master.

Signed-off-by: Sebastian Krah <skrah@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by tspmelo** [comment]
Can't we keep the translations from master? All the translations are lost in the locales files.

File: src/pybind/mgr/dashboard/frontend/src/locale/messages.cs.xlf [hunk 1]
```diff
@@ -758,154 +944,155 @@
...
       </trans-unit>
       <trans-unit id="9bd96fcf50863e685c74d0490392f46689ffbdb6" datatype="html">
         <source>Required field</source>
-        <target>Vyžadovaná kolonka</target>
+        <target>Required field</target>
       </trans-unit>
       <trans-unit id="a5f3f74c0f6925826cb2188576391c0da01a23f0" datatype="html">
         <source>Must contain one or more comma-separated values</source>
-        <target>Je třeba, aby obsahovalo jednu nebo více čárkou oddělovaných hodnot</target>
+        <target>Must contain one or more comma-separated values</target>
```
Comment (Devp00l) [inline src/pybind/mgr/dashboard/frontend/src/locale/messages.cs.xlf:983]: I'm not sure if this sentence should really be replaced with English. Please correct me if I'm wrong, which is probably the case with my wild guess ;)

File: src/pybind/mgr/dashboard/frontend/src/locale/messages.de-DE.xlf [hunk 2]
```diff
@@ -3,7 +3,7 @@
     <body>
       <trans-unit id="319e0745bcbc132451569294fa2fa21bf10f555a" datatype="html">
         <source>Toggle navigation</source>
-        <target>Navigation umschalten</target>
+        <target>Toggle navigation</target>
```
Comment (Devp00l) [inline src/pybind/mgr/dashboard/frontend/src/locale/messages.de-DE.xlf:6]: Same here

File: src/pybind/mgr/dashboard/frontend/src/locale/messages.zh-CN.xlf [hunk 3]
```diff
@@ -3,75 +3,71 @@
     <body>
       <trans-unit id="319e0745bcbc132451569294fa2fa21bf10f555a" datatype="html">
         <source>Toggle navigation</source>
-        <target>切换导航</target>
+        <target>Toggle navigation</target>
```
Comment (Devp00l) [inline src/pybind/mgr/dashboard/frontend/src/locale/messages.zh-CN.xlf:6]: It seems to me like if all language files now contain the same strings (I haven't compared them with `diff`). I'm nearly convinced by now that something is wrong.

**Comment by Exotelis** [comment]
@tspmelo I already noticed that and I'm working on a fix. Thanks for pointing this out

**Comment by yuriw** [comment]
[wip-yuri3-testing-2020-04-14-1612-octopus](https://trello.com/c/eaAX6eg4/865-wip-yuri3-testing-2020-04-14-1612-octopus)

**Pull request merged**